packages/lending/src/protocols/aave-v2/lending-protocol.ts

```diff
@@ -186,6 +194,8 @@ export class LendingProtocol extends Protocol {
             calculateCompoundedRate({ rate: variableBorrowRate.toString(), duration: SECONDS_PER_YEAR }),
             RAY_DECIMALS
           ),
+          totalSupply: common.toBigUnit(totalSupply, asset.decimals),
```

Comment:
`totalSupply` is from aToken, should we use `aToken.decimals` as `decimals` here? (the same as aavev3 and radiantv2)

Reply:
OK~

Reply:
As discussed, maintain the original underlying decimals